test/specs/elements/Input/Input-test.js

```diff
@@ -170,42 +186,39 @@ describe('Input', () => {
     })
   })
 
+  describe('ref', () => {
+    it('maintains ref on child node', () => {
+      const ref = sandbox.spy()
+      const wrapper = mount(<Input><input ref={ref} /></Input>)
+
+      // ref.should.have.been.calledOnce()
```

Comment:
Refs are not passed through props. React makes them undefined. We'll have to accept the ref on a different prop name, or, update factories to include refs in the overrides. We should probably use a different prop name, such as, `inputRef`.